validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/Eth1DataCache.java

```diff
@@ -72,7 +72,7 @@ public void onTick(Date date) {
     prune();
   }
 
-  // Called by ValidatorCoordinator not the event bus to ensure we process slot events in sync
+  // Called by BeaconChainController not the event bus to ensure we process slot events in sync
```

Comment:
I'm unsure if the onSlot method here does need to be called by BeaconChainController or not, but will leave it for another PR, since I want to merge this in sooner rather than later.

Reply:
Yeah, not exactly sure where it should fit in - it's internal to the beacon chain so probably shouldn't be using the event channel but not sure.  Ideally I think we'd just get rid of it but will need to think through the pruning a bit more and see where it lands.